src/structures/Chat.js

```diff
@@ -177,6 +177,20 @@ class Chat extends Base {
         }, this.id._serialized);
     }
 
+    /**
+     * Opens the Chat Window of the chat
+     */
+    async openWindow() {
+        return this.client.interface.openChatWindow(this.id._serialized);
+    }
+
+    /**
+     * Opens the Chat Drawer of the chat
+     */
+    async openDrawer() {
+        return this.client.interface.openChatDrawer(this.id._serialized);
+    }
+    
```

Comment:
Everything else looks fine, but I would remove these to keep all interface-related things in the interface controller. 

```suggestion
```